docs/design/access.md

```diff
@@ -119,7 +119,7 @@ Related discussion:
 
 This doc describes two security profiles:
   - Simple profile:  like single-user mode. Make it easy to evaluate K8s
-without lots of configuring accounts and policies.  Protects from unauthorized
+without lots of configuring accounts and policies.  Protect from unauthorized
```

Comment:
While it is an incomplete sentence, "Protects from" has an implied "it" or "this" subject.